docs/book/src/developer/providers/v1.2-to-v1.3.md

```diff
@@ -42,6 +42,7 @@ in Cluster API are kept in sync with the versions used by `sigs.k8s.io/controlle
 The default value is 0, meaning that the volume can be detached without any time limitations.
 - A new annotation `machine.cluster.x-k8s.io/exclude-wait-for-node-volume-detach` has been introduced that allows explicitly skip the waiting for node volume detaching.
 - The `Path` func in the `sigs.k8s.io/cluster-api/cmd/clusterctl/client/repository.Overrider` interface has been adjusted to also return an error.
+- Two new flags have been introduced to expose the configuration of the TLS Options for the webhook server, namely `--tls-min-version` and `--tls-cipher-suites`. Utility functions have been exposed under the `util/flags` package which can be used by the providers to implement the similar functionality.
```

Comment:
Should this be moved under other? (it isn't an API change)
Also, if we want providers to implement those flags let's add a new section "required (or suggested) changes for providers" with something like

- Provider MUST / SHOULD (depending by what we choose) expose the configuration of the TLS Options for the webhook server; it is recommended to use utility functions under the `util/flags` package to ensure consistency with CAPI/other providers

Reply:
Moving this out to a different PR, will make the changes there.

Reply:
Checkout https://github.com/kubernetes-sigs/cluster-api/pull/7511